app/javascript/src/components/Common/Menubar.jsx

```diff
@@ -0,0 +1,75 @@
+// @ts-nocheck
+import React, { useState } from "react";
+
+import { Search, Settings, Plus } from "neetoicons";
+import { Typography } from "neetoui";
+import { MenuBar } from "neetoui/layouts";
+
+function Menubar({ title, setCategory, options }) {
+  const [isSearchCollapsed, setIsSearchCollapsed] = useState(true);
+
+  return (
+    <MenuBar showMenu={true} title={title}>
+      {options?.heads?.map((head, idx) => (
+        <MenuBar.Block
+          label={head.title}
+          key={idx}
+          count={head.count}
+          onClick={() => setCategory(head.title)}
+        />
+      ))}
+
+      <MenuBar.SubTitle
+        iconProps={[
+          {
+            icon: Search,
+            onClick: () => setIsSearchCollapsed(!isSearchCollapsed),
```

Comment:
Please carefully read: https://www.bigbinary.com/react-best-practices/know-when-to-use-previous-state-for-state-update